gravitee-apim-gateway/gravitee-apim-gateway-services/gravitee-apim-gateway-services-debug/src/main/java/io/gravitee/gateway/debug/policy/impl/DebugPolicyFactory.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright © 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.gateway.debug.policy.impl;
+
+import io.gravitee.gateway.policy.PolicyPluginFactory;
+import io.gravitee.gateway.reactive.core.condition.ExpressionLanguageConditionFilter;
+import io.gravitee.gateway.reactive.policy.ConditionalPolicy;
+import io.gravitee.gateway.reactive.policy.DefaultPolicyFactory;
+
+public class DebugPolicyFactory extends DefaultPolicyFactory {
```

Comment:
you can add javadoc explaining why we need this class that seems to do nothing (to differentiate its type from DefaultPolicyFactory)

Reply:
Addressed the request.